### PR TITLE
L man's PR

### PR DESCRIPTION
**Pull request by hamzafj5**
Added my piece of code in the cpp file

### CODE REVIEW

**Review by haseebuchiha — CHANGES_REQUESTED** [review]
No update for link list

File: CustomeDB/CustomeDB.cpp [hunk 1]
```diff
@@ -94,3 +94,96 @@ int main()
 
     return 0;
 }
+/////////////////////////////////////////////////////////////////////
+//Hamza was here L
+#include <iostream>
+#include <string>
+using namespace std;
+
+struct Node {
```
Comment (Mimi848) [inline CustomeDB/CustomeDB.cpp]: there is memory leakage in your program. i want u to find it out. for hint run add data and delete at head hundreds of thousands of time try testing it.

**Review by Mimi848 — CHANGES_REQUESTED** [review]
for(int i = 0; i < 10000000 ; i++){
        l1.addAtEnd(1, "talha", 20);
        l1.addAtEnd(2, "haseed", 10);
        l1.addAtEnd(3, "bats", 25);
        l1.deleteAtHead();
        l1.deleteAtHead();
        l1.deleteAtHead();
        
    }

**Review by haseebuchiha — CHANGES_REQUESTED** [review]
Same issues as talha. Definitions should be written in class.cpp files

**Pull request closed without merging**